hack/install-github-release.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
```

Comment:
How is this change related to the rest? Would you please split the PR into multiple commits (or multiple PRs, but won't fight for it). Moving to newer operator-sdk, go.mod, this little script.

Reply:
We cannot, proper support for operator-sdk is done at 0.12.0 so it have to be at the same PR, although I have to remove this script.

Reply:
You promised to remove this :(

Reply:
Broken promises, let's remove it :-)